databricks/koalas/config.py

```diff
@@ -130,7 +132,6 @@ def validate(self, v: Any) -> None:
             lambda v: v is None or v >= 0,
             "'compute.max_rows' should be greater than or equal to 0.")),
 
-    # TODO: None should support unlimited.
```

Comment:
`shortcut_limit` cannot support `None` as at least it needs empty dataframe to infer its type when its needed.